test/components/DayPickerRangeController_spec.jsx

```diff
@@ -741,6 +744,43 @@ describe('DayPickerRangeController', () => {
               const minimumNightsCalls = getCallsByModifier(addModifierToRangeSpy, 'blocked-minimum-nights');
               expect(minimumNightsCalls.length).to.equal(0);
             });
+
+            it('updates state to remove `blocked-minimum-nights` and `blocked` from the appropriate days', () => {
+              const startDate = today;
+              const minimumNights = 5;
+              const wrapper = shallow(<DayPickerRangeController
+                {...props}
+                focusedInput={END_DATE}
+                startDate={startDate}
+                minimumNights={minimumNights}
+              />);
+              const { visibleDays } = wrapper.state();
+              let day = moment(today);
+              for (let i = 0; i < minimumNights; i += 1) {
+                const monthString = toISOMonthString(day);
+                const dateString = toISODateString(day);
+                expect(visibleDays[monthString][dateString]).to.include('blocked-minimum-nights');
+                expect(visibleDays[monthString][dateString]).to.include('blocked');
+                day.add(1, 'day');
+              }
+
+              wrapper.instance().componentWillReceiveProps({
```

Comment:
btw i think this might be covered by `wrapper.setProps`? not a blocker tho

Reply:
This is what I've been using in all the tests in this describe block, so if I wanted to update I'd probably do everything all at once at a later date. :)